api/project/models/project.js

```diff
@@ -5,4 +5,30 @@
  * to customize this model
  */
 
-module.exports = {};
+module.exports = {
+  /**
+  * Triggered before user creation.
+  */
+  lifecycles: {
```

Comment:
What is lifecycles?

Reply:
It says "The lifecycle hooks are functions that get triggered when the Strapi queries are called. They will get triggered automatically when you manage your content in the Admin Panel or when you develop custom code using queries.", it is triggered after the project is created, so we can create the google meeting and pass in ProjectID.